### ISSUE TITLE
TinyMCE rich text controls have no label

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#1723]
From user: "TinyMCE (or maybe Orbeon) inserts a span with class 'mceVoiceLabel' containing the text 'Rich Text Area'. Our test user heard this read out when inspecting the form. If it could be replaced with the label text specified for that control, this might be a solution."

See also [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute).
